Week3/Hotel.sql

```diff
@@ -0,0 +1,96 @@
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `mydb` DEFAULT CHARACTER SET utf8 ;
+USE `mydb` ;
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Employees`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Employees` (
+  `idEmployees` INT NOT NULL,
+  `username` VARCHAR(45) NOT NULL,
+  `Password` VARCHAR(45) NULL,
+  PRIMARY KEY (`idEmployees`),
+  UNIQUE INDEX `username_UNIQUE` (`username` ASC))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`customer`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`customer` (
+  `idcustomer` INT NOT NULL AUTO_INCREMENT,
+  `first name` VARCHAR(45) NOT NULL,
```

Comment:
It's better not to use spaces in column names, if you try to query this you will always have to put back quotes around the words 'first name' for the select statement to work. It will assume the column name is done after you write the word `first`. Try it on your database.